src/Documentation/DocumentationGenerator/DocumentationWriter.cs

```diff
@@ -149,10 +149,13 @@ public async Task WriteOutput(QsNamespace ns, DocComment docComment)
                 // DocFX metadata
                 ["uid"] = name,
                 ["title"] = title,
+                ["author"] = "msquantumdocs",
```

Comment:
This may be a problem, since the new documentation generation pipeline is also intended to be used by external users as well as internal users. If we want to attach author metadata, we should do so in a way that's configurable so as to allow external users to set themselves as authors.